### ISSUE TITLE
Error while installing nw-builder

### ISSUE THREAD

**Issue by Yadav420dilip** [nwutils/nw-builder#530]
command: npm i nw-builder -D
node version: v12.18.0
npm version: 6.14.4

**Error:**
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno ENOENT
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/adam-lynch/node-temp.git
npm ERR! enoent 
npm ERR! enoent 
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2020-06-10T08_05_49_610Z-debug.log

**Comment by ZenbladeJS** [comment]
same problem here

**Comment by Giwrgos-Leivadias** [comment]
Me too

**Comment by ToastyMallows** [comment]
I'm not getting this error locally but I'm getting it while running a Github Action:

```
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\bin\git.EXE ls-remote -h -t ssh://git@github.com/adam-lynch/node-temp.git
npm ERR! 
npm ERR! Host key verification failed.
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR! 
npm ERR! exited with error code: 128
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\npm\cache\_logs\2020-12-27T20_32_02_823Z-debug.log
```

**Comment by MBurtonshaw** [comment]
I'm also getting it

**Comment by applefell** [comment]
make sure you have git installed and try again

**Comment by djmisterjon** [comment]
sometime it a server problem when fetching the zip.
It will stop before complete and continue.
Wait and restart.
Sometime it take me ~3 time before succes.
clear cache in node module can help if the partial zip file downloaded are corrupt.

Maybe need a script that's allow low latency of server.

**Comment by ayushmxn** [comment]
https://github.com/nwutils/nw-builder/issues/521#issuecomment-1139692576

**Comment by ayushmxn** [comment]
Fixed by #583
